doc/tutorials/11-ferrofluid/11-ferrofluid_part1.ipynb

```diff
@@ -88,13 +88,13 @@
     "\n",
```

Comment:
<p>all constants should be CAPITAL. The single character variable name for the number of particles should be changed to N_PART. The verbosity of a variable name should roughly correspond to the size it's scope. dt is probably also a candidate (-&gt; TIME_STEP)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/espressomd/espresso/pull/3919/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>